src/Caller/Masa.Utils.Caller.Core/Internal/DefaultTypeConvertProvider.cs

```diff
@@ -0,0 +1,109 @@
+﻿namespace Masa.Utils.Caller.Core.Internal;
+
+public class DefaultTypeConvertProvider : ITypeConvertProvider
+{
+    private static readonly ConcurrentDictionary<Type, List<PropertyInfoMember>> Dictionary = new();
+
+    protected readonly List<Type> BasicDataTypes = new()
+    {
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.type.isprimitive?view=net-6.0
Is it not enough?

Reply:
IsPrimitive is not enough, simplifies the serialization judgment